fiftyone/utils/torch.py

```
@@ -1524,187 +1665,147 @@
+            faster data loading times, but you must have enough memory to store
+            the required field values for the entire collection. When
+            ``vectorize=True``, all field values must be serializable; ie
+            ``pickle.dumps(field_value)`` must not raise an error
+        skip_failures (False): whether to skip failures that occur when calling
+            ``get_item``. If True, the exception will be returned rather than
+            the intended field values
+        local_process_group (None): the local process group. Only used during
+            distributed training
     """
 
     def __init__(
```

Pylint (3.3.7)

[refactor] Too many positional arguments (6/5)

(R0917)